aptoide-installer/src/main/java/cm/aptoide/pt/installer/platform/File.kt

```diff
@@ -0,0 +1,41 @@
+package cm.aptoide.pt.installer.platform
+
+import android.text.TextUtils
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.io.IOException
+import java.math.BigInteger
+import java.security.MessageDigest
+import java.security.NoSuchAlgorithmException
+
+
```

Comment:
The file could have a better name, like 'FileExtensions' for example

Reply:
Fixed and force pushed